hpx/runtime/actions/basic_action.hpp

```diff
@@ -62,14 +62,12 @@ namespace hpx { namespace actions
             HPX_MOVABLE_BUT_NOT_COPYABLE(continuation_thread_function);
 
         public:
-            template <typename F_, typename ...Ts_>
             explicit continuation_thread_function(
-                    std::unique_ptr<continuation> cont,
-                    naming::address::address_type lva, F_&& f, Ts_&&... vs)
+                std::unique_ptr<continuation> cont,
+                naming::address::address_type lva, F&& f, Ts&&... vs)
```

Comment:
That's correct, this constructor does not do deduction anymore (but it is still forwarding). `F` and `Ts` are the exact types deduced earlier, there is no need to re-deduce them as the only cause for a mismatch would be bad code (which we'd now see at compile time).